### PR TITLE
💃 Added Customizable Messages :Man:

### PR DESCRIPTION
**Pull request by ChopChoP534**
read readme uwu

### CODE REVIEW

**Comment by Assassin-1234** [comment]
nice! thanks for contributing :)

can you please move the baseOptions function to Giveaways class tho?

**Comment by Assassin-1234** [comment]
> nice! thanks for contributing :)
> 
> can you please move the baseOptions function to Giveaways class tho?

or just refactor the func name to something more relatable

**Comment by ChopChoP534** [comment]
done :)

**Comment by ChopChoP534** [comment]
> > nice! thanks for contributing :)
> > can you please move the baseOptions function to Giveaways class tho?
> 
> or just refactor the func name to something more relatable

baseMessages? defaultMessages? messages?
u tell me

**Comment by Assassin-1234** [comment]
> > > nice! thanks for contributing :)
> > > can you please move the baseOptions function to Giveaways class tho?
> > 
> > 
> > or just refactor the func name to something more relatable
> 
> baseMessages? defaultMessages? messages?
> u tell me

GiveawayMessages sounds good

**Comment by Assassin-1234** [comment]
or wait if its in giveaways class then only messages

**Review by Assassin-1234 — CHANGES_REQUESTED** [review]
everything looks good just change the func name and leave the conficts, I'll resolve them tomorrow morning

**Review by ChopChoP534 — COMMENTED** [review]
👍

**Review by Assassin-1234 — APPROVED** [review]

**Review by Assassin-1234 — APPROVED** [review]

**Review by pro-gamer007 — CHANGES_REQUESTED** [review]
Everything is good except from the previous comment i left on line 56 in src/classes/giveaways.js

File: src/classes/giveaways.js [hunk 1]
```diff
@@ -28,7 +53,7 @@ class giveaways {
 		if (!endAfter) throw new Error('NuggiesError:  time wasnt provided while creating giveaway');
 		if (typeof endAfter !== 'string') throw new TypeError('NuggiesError: endAfter should be a string');
 		if (!channel) throw new Error('NuggiesError: channel wasnt provided while creating giveaway');
-		const msg = await message.guild.channels.cache.get(channel).send('', {
+		const msg = await message.guild.channels.cache.get(channel).send(message.client.giveawayMessages.giveaway, {
```
[the remark this answers is not in the document]
Comment (pro-gamer007) [reply src/classes/giveaways.js:56]: my bad, didnt see it properly 😳

**Review by pro-gamer007 — APPROVED** [review]

**Review by LegItMate — APPROVED** [review]

**Pull request merged**